### PR TITLE
Cli

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: test/logr.test.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ describe('logr', () => {
         }
       });
       log(['tag1'], 'message');
-      expect(lastMessage).to.equal('[\u001b[31mtag1\u001b[0m] message');
+      expect(lastMessage).to.equal('[\u001b[41mtag1\u001b[0m] message');
```
Comment (jgallen23) [inline test/logr.test.js]: why did this change?

File: test/logr.test.js [hunk 2]
```diff
@@ -148,7 +148,7 @@ describe('logr', () => {
         }
       });
       log(['error', 'warn', 'notice'], 'message');
-      expect(lastMessage).to.equal('[\u001b[31merror\u001b[0m,\u001b[33mwarn\u001b[0m,\u001b[34mnotice\u001b[0m] message\u0007');
+      expect(lastMessage).to.equal('[\u001b[41merror\u001b[0m,\u001b[43mwarn\u001b[0m,\u001b[44mnotice\u001b[0m] message\u0007');
```
Comment (jgallen23) [inline test/logr.test.js]: and this?

File: lib/colors.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+module.exports ={
+  reset: '\x1b[0m',
+  bright: '\x1b[1m',
+  dim: '\x1b[2m',
+  underscore: '\x1b[4m',
+  blink: '\x1b[5m',
+  reverse: '\x1b[7m',
+  hidden: '\x1b[8m',
+  fgBlack: '\x1b[30m',
```
Comment (jgallen23) [inline lib/colors.js]: why did the names of the colors change?

File: test/logr.test.js (hunk 1 above)
Comment (jgallen23) [inline test/logr.test.js]: I see why now.  we shouldn't change the color names. We should just change the defaults in index.js

**Comment by jgallen23** [comment]
just a reminder to have each feature in a separate PR. looks like two got mixed in here

File: lib/cli.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+const stringify = require('json-stringify-safe');
+const _ = require('lodash');
+const colors = require('./colors');
+
+module.exports = function(options, tags, message) {
+  if (typeof message === 'object') {
+    message = stringify(message, null, (options.pretty) ? '  ' : '').replace(/\\n/g, '\n');
+  }
+  if (options.consoleBell && (_.intersection(options.consoleBell, tags).length > 0)) {
```
Comment (jgallen23) [inline lib/cli.js]: no console bell for cli

File: lib/cli.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    message += '\u0007';
+  }
+
+  if (options.colors) {
```
Comment (jgallen23) [inline lib/cli.js]: don't need this for cli

File: lib/cli.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    tags.forEach((tag, i) => {
+      if (options.colors[tag]) {
+        tags[i] = `${colors[options.colors[tag]]}${tag}${colors.reset}`;
+      }
+    });
+  }
+  if (options.lineColor) {
+    message = `${colors[options.lineColor]}  ${message}`;
```
Comment (jgallen23) [inline lib/cli.js]: I think you need to reset the color after the message

**Comment by orthagonal** [comment]
@jgallen23 should be ready to have master merged into it

**Pull request merged**